visualdl/logic/CMakeLists.txt

```diff
@@ -34,3 +34,12 @@ else()
   target_link_libraries(core PRIVATE pybind entry python im tablet storage sdk protobuf ${OPTIONAL_LINK_FLAGS})
 endif()
 set_target_properties(core PROPERTIES PREFIX "" SUFFIX ".so")
+
+add_custom_command(TARGET core POST_BUILD
+  COMMAND "${CMAKE_COMMAND}" -E copy
```

Comment:
why `copy` needed here, there is a `copy core.so` in `setup.py`, is this duplicate to that?

Reply:
@Superjomn 
I have a 2 new CMAKE targets "visualdl_docs_en" and "visualdl_docs_cn".  These don't use setup.py (or build.sh), which copies the core.so files to visualdl and visualdl/python directories.   I think we can remove the copy core.so from "build.sh", or I can put a "if(WITH_DOC)" around this cmake.

Note, I need to build core.so so Sphinx can generate Python API documentation